src/js/relayclient/EIP712/RelayData.js

```diff
@@ -0,0 +1,10 @@
+class RelayData {
```

Comment:
what the point? its a struct that gets used exactly once. separating into a file requires more lines, more initialization, more source for bugs...

Reply:
I love files, and types, and lines